test/ui/selenium/LoginTests.js

```diff
@@ -0,0 +1,252 @@
+const { Assertion } = require('chai');
```

Comment:
Can the imports be typescript and not javascript as the project is in typescript

Reply:
done